### PR TITLE
Desktop preview

### PR DESCRIPTION
**Pull request by joshualyngdoh**
# Description
- Added media query
- Checked for linter errors
- Set up Desktop version

# Screenshots
- 
image

image

image

image

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -416,3 +408,179 @@ textarea {
...
+
+  .vivo h5 {
+    padding-left: 12px;
+  }
+
+  .footer-background {
+    display: flex;
+    align-items: center;
+    justify-items: center;
+  }
```
Comment (brainconnect93) [inline styles.css]: - [ ] Hello @joshualyngdoh, I must commend that you have done wonderful work on your project and you made a great effort in getting the desktop version of the project done 👏 👏 🌹 . But I would kindly suggest we stick to the Figma template for the desktop version as much as possible. Currently, the `contact` section of your project is missing the `background-image` as we have it in the Figma template, Kindly see to fixing it before re-submitting for review

Here is the Figma design 👇 

cont

Your Design 👇 

josh

- [ ] Kindly work on the `border-radius` around the `input` form too, to look perfect like what we have in the Figma design.

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @joshualyngdoh, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (joshualyngdoh) [reply styles.css]: Thank you. I will make all the changes and request a new review.

**Review by roshan-bajgain — APPROVED** [review]
Hi @joshualyngdoh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**